libnd4j/include/ops/declarable/generic/parity_ops/reduce_prod.cpp

```diff
@@ -93,8 +89,21 @@ namespace ops {
             std::unique_ptr<ResultSet<T>> inList(NDArrayFactory<T>::allTensorsAlongDimension(input, dimensions));
             //output->
             for (Nd4jLong e = 0; e < outList->size(); ++e) {
+//                auto backpropRoutine = LAMBDA_T(_x, epsilon, tempProd, e) {
+//                    return (*epsilon)(e) * _x / (*tempProd)(e);
+//                };
+                outList->at(e)->assign(epsilon);
+                outList->at(e)->template applyPairwiseTransform<simdOps::Multiply<T>>(tempProd, nullptr);
+                outList->at(e)->template applyPairwiseTransform<simdOps::Divide<T>>(inList->at(e), nullptr);
+//                (*outList->at(e)) *= (*inList->at(e));
+//                (*outList->at(e)) /= (*tempProd)(e);
+                
                 //outList->at(e)->assign(epsilon);
```

Comment:
This is, effectively loop with no code within. And lots of comments.

What's the point here?